packages/commercetools/theme/components/Checkout/ShippingForm.vue

```diff
@@ -284,10 +284,10 @@ export default {
       return Boolean(addresses?.length);
     });
 
-    const handleStepSubmit = () => context.emit('stepSubmit');
+    const handleStepSubmit = () => context.emit('submit');
```

Comment:
its redundant to wrap single function into another function

Reply:
I will remove this part from this PR, as https://github.com/vuestorefront/vue-storefront/pull/5564 takes care of that